themes/sgsc/sass/_about.scss

```diff
@@ -1,11 +1,19 @@
 .page-template-page-about {
+  .entry-content {
+    padding: 0;
+    margin: 0;
+    margin-top: 1rem;
```

Comment:
Can you make this margin smaller?

Reply:
Unless it's changing this margin on the About page is equal to that of the Programs page.


About:
![image](https://user-images.githubusercontent.com/26963647/33303427-fc3da2e0-d3b7-11e7-996a-a28dc89c45f4.png)

Programs:
![image](https://user-images.githubusercontent.com/26963647/33303450-171834f4-d3b8-11e7-9464-2e9a061154d8.png)

Reply:
![image](https://user-images.githubusercontent.com/26846510/33304358-1d6eb3d2-d3bd-11e7-8ad7-4c14d7e6fb2f.png)

Reply:
The proportions of everything should be like programs